T-Dict/T-Dict/Extension/String.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  String.swift
+//  T-Dict
+//
+//  Created by Tiến on 3/29/21.
+//  Copyright © 2021 Tiến. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    public static func safeString(string: String) -> String {
+        let safeString = string.trimmingCharacters(in: .whitespacesAndNewlines).replacingOccurrences(of: " ", with: "%20")
+        return safeString
+    }
```

Comment:
static là không cần thiết
dùng luôn extension cho chính String đó
```suggestion
    var safeString: String {
        return trimmingCharacters(in: .whitespacesAndNewlines).replacingOccurrences(of: " ", with: "%20")
    }
```

Reply:
`"ABCDXYZ".safeString`